itou/siaes/models.py

```diff
@@ -636,6 +643,29 @@ def clean(self):
                 }
             )
 
+    @classmethod
+    def from_db(cls, db, field_names, values):
+        # this is close to the code we have in itou.users.User.from_db() but we
+        # don't want full genericity yet. We could be DRYer by using a mixin later,
+        # but we'd need to handle some extra edge cases.
+        instance = super().from_db(db, field_names, values)
+        setattr(instance, "_old_is_active", instance.is_active)
+        return instance
+
+    def save(self, *args, **kwargs):
+        if hasattr(self, "_old_is_active") and self._old_is_active != self.is_active:
+            self.field_history.append(
+                {
+                    "field": "is_active",
+                    "from": self._old_is_active,
+                    "to": self.is_active,
+                    "at": timezone.now(),
+                }
+            )
+            if "update_fields" in kwargs and "field_history" not in kwargs["update_fields"]:
```

Comment:
Pour ma culture, le `"field_history" not in` c'est toi ou c'est Django qui n'aime pas avoir 2 fois le même nom de champ ?